properties/P000058.md

```diff
@@ -1,10 +1,14 @@
 ---
 uid: P000058
-name: Smaller than the continuum
+name: Cardinality $\lt\mathfrak c$
+aliases:
+  - Smaller than the continuum
+  - Cardinality $\lt\beth_0$
```

Comment:
I don't know about this.  Do we really need to mention beth numbers here?

Reply:
I'm inclined to keep aleph/beth stuff; it wasn't necessary today, but I added it in while I was working. In particular, I never want to write $2^{2^{2^{\mathfrak c}}}$ and we certainly might want $\aleph_1,\aleph_2,\dots$ someday. By putting in these aliases, we get relevant properties appearing in search queries.